spec/person_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'json'
+require_relative '../person'
+
+describe 'Person' do
+  before(:each) do
+    @person = Person.new(22, 'TestPerson', parent_permission: false)
+  end
+
+  it 'Should take the age, the name, and the parent_permission' do
+    expect(@person.age).to eq(22)
+    expect(@person.name).to eq('TestPerson')
+    expect(@person.parent_permission).to eq(false)
+  end
+
+  it 'Should take the age and set default values' do
+    new_person = Person.new(28)
+    expect(new_person.age).to eq(28)
+    expect(new_person.name).to eq('Unknown')
+    expect(new_person.parent_permission).to eq(true)
+  end
+
+  it 'Should return true if the age is greater than or equal to 18' do
```

Comment:
- [ ] @team great work implementing the changes, your description in line 25 and line 30 should be interchanged so that the it statement matches with the result in your test.